packages/browsers/test/src/versions.ts

```diff
@@ -6,6 +6,6 @@
 
 export const testChromeBuildId = '127.0.6533.72';
 export const testChromiumBuildId = '1083080';
-export const testFirefoxBuildId = '130.0a1';
+export const testFirefoxBuildId = '131.0a1';
```

Comment:
@Lightning00Blade should we just pin some stable version and remove this test update job?

Reply:
Are stable versions always present? If so yet's do that.

Reply:
yes, they should be, this was only a workaround for nightly that get overwritten

Reply:
I will open a PR